files/build_templates/sonic_debian_extension.j2

```diff
@@ -61,7 +61,7 @@ sudo bash -c "echo \"DOCKER_OPTS=\"--storage-driver=overlay2\"\" >> $FILESYSTEM_
 # Copy docker start script to be able to start docker in chroot
 sudo mkdir -p "$FILESYSTEM_ROOT/$DOCKER_CTL_DIR"
 sudo cp $DOCKER_SCRIPTS_DIR/docker "$FILESYSTEM_ROOT/$DOCKER_CTL_SCRIPT"
-if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+if [ $EMULATED_COMPILE == y ] && [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
```

Comment:
i do not see why you need 2nd and 3rd condition here.

Reply:
I keeped 2nd and 3rd condition here is just for keeping old logic unchanged. I agree with you and will reomve these 2 conditions. Other related comments will perform the same modify.